hw/ip/aes/pre_sca/alma/cpp/verilator_tb_aes_sbox.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdio.h>
+
+#include "Vcircuit.h"
+#include "testbench.h"
+
+int main(int argc, char **argv) {
+  Verilated::commandArgs(argc, argv);
+  Testbench<Vcircuit> *tb = new Testbench<Vcircuit>();
```

Comment:
You don't really need a pointer here. Doing it like this will get rid of all the `->`'s:
```suggestion
  Testbench<Vcircuit> tb;
```

Reply:
I would have thought so, too. However, removing the pointer and adjusting the remaining code accordingly causes the VCD not getting written to disk. I have no idea why but I would suggest to just leave this code as is then.

Reply:
Hmm, ok I guess. It's a bit weird because your current code actually leaks `tb` and never runs the destructor. I'm surprised that this is more likely to write the output than the version that destroys things correctly.

Reply:
I've had another look at this. Needed to initialize the `m_trace` pointer inside `tb` to `NULL` for the traces to get written to disk. Now switched `*tb` to `tb` as you suggested.